util/flags.py

```diff
@@ -118,7 +118,7 @@ def create_flags():
     f.DEFINE_boolean('show_progressbar', True, 'Show progress for training, validation and testing processes. Log level should be > 0.')
 
     f.DEFINE_boolean('log_placement', False, 'whether to log device placement of the operators to the console')
-    f.DEFINE_integer('report_count', 10, 'number of phrases with lowest WER(best matching) to print out during a WER report')
+    f.DEFINE_integer('report_count', 5, 'number of phrases with best WER, median WER and with worst WER to print out during a WER report')
```

Comment:
nit: unclear to the user here whether we will have 5 sentences covering all best, median and worst, or if we have 5 of each